packages/app/cypress/e2e/integration/sidebar_navigation.spec.ts

```diff
@@ -6,6 +8,10 @@ describe('Sidebar Navigation', () => {
     cy.visitApp()
   })
 
+  it('expands the left nav bar by default', () => {
+    cy.get('[aria-expanded]').should('have.attr', 'aria-expanded', 'true')
```

Comment:
I think this locator is too broad. `[aria-expanded]` could match multiple things on the page (currently it matches 3, because some of the buttons have it), and if any of them is expanded, this will pass. A good way to scope it to the intended area might be to base it off the accessible name of the sidebar, which is probably not going to conflict:

```suggestion
    cy.findByLabelText('Sidebar').closest('[aria-expanded]').should('have.attr', 'aria-expanded', 'true')
```

Looks like there are a couple of tweaks needed to make this expand/contract behavior more accessible.